data/Music/Sonic the Hedgehog Series/Sonic the Hedgehog 1/Fanfare - 1Up.meta

```diff
@@ -1,3 +1,4 @@
 Sonic the Hedgehog - 1-Up
 0x16
-Fanfare
\ No newline at end of file
+Fanfare
+HeartContainerGet,HeartPieceGet,EventFanfare
```

Comment:
The location line should have at least one group from each level of specificity (exact, high, mid and low).
```suggestion
HeartContainerGet,HeartPieceGet,ItemFanfare,EventFanfare
```